### PR TITLE
WKB write option added

### PR DESCRIPTION
**Pull request by gc-sommer**
With this pull request you can also convert any rasterio readable raster format to WKB raster

### CODE REVIEW

File: wkb_raster.py [hunk 1]
```diff
@@ -214,3 +217,203 @@ def read_wkb_raster(wkb):
...
+            # |               |              | 5: 16-bit signed integer          |
+            # |               |              | 6: 16-bit unsigned signed integer |
+            # |               |              | 7: 32-bit signed integer          |
+            # |               |              | 8: 32-bit unsigned signed integer |
+            # |               |              | 9: 32-bit float                   |
+            # |               |              | 10: 64-bit float                  |
+            # +---------------+--------------+-----------------------------------+
+
+            # not used - always False
+            ifOffline = False
```
Comment (nathancahill) [inline wkb_raster.py]:
```suggestion
            isOffline = False
```

File: wkb_raster.py [hunk 2]
```diff
@@ -214,3 +217,203 @@ def read_wkb_raster(wkb):
...
+                      'u2', 'i4', 'u4', 'f4', 'f8']
+
+            rasterio_dtype = src.meta.get("dtype")
+            dt_short = np.dtype(rasterio_dtype).descr[0][1][1:]
+            pixtype = dtypes.index(dt_short)
+
+            fmt = fmts[pixtype]
+
+            # format binary -> :b
+            binary_str = f"{ifOffline:b}{hasNodataValue:b}{isNodataValue:b}{reserved:b}{pixtype:b}"
```
Comment (nathancahill) [inline wkb_raster.py]:
```suggestion
            binary_str = f"{isOffline:b}{hasNodataValue:b}{isNodataValue:b}{reserved:b}{pixtype:b}"
```

File: wkb_raster.py [hunk 3]
```diff
@@ -214,3 +217,203 @@ def read_wkb_raster(wkb):
...
+
+            # unset
+            reserved = False
+
+            # band['isOffline'] = bool(bits & 128)  # first bit
+            # band['hasNodataValue'] = bool(bits & 64)  # second bit
+            # band['isNodataValue'] = bool(bits & 32)  # third bit
+
+            # pixtype = bits & 15  # bits 5-8
+
```
Comment (nathancahill) [inline wkb_raster.py]: Remove.

```suggestion
```

**Comment by gc-sommer** [comment]
@nathancahill thanks for your review! I forgot to bump up the version number in setup.py - could you change this?

**Pull request merged**